unah254/SCA-devops
# Screenshots

#### What does this PR do?
Adds screenshots of commands I ran on terminal
#### Description of Task to be completed?
- Add a folder containing all screenshots
#### How should this be manually tested?
- Clone the repository and cd into it
-  Checkout to branch `git checkout screenshots`
-  From the editor, you should be able to view a folder named screenshots containing images
#### Any background context you want to provide?
None
#### Questions:
None
